src/pages-and-resources/discussions/data/api.js

```diff
@@ -57,6 +66,11 @@ function normalizeApps(data) {
     },
     activeAppId: data.providers.active,
     apps,
+    discussionTopicIds: Object.entries(
+      data.plugin_configuration.discussion_topics,
+    // eslint-disable-next-line no-unused-vars
+    ).map(([key, value]) => value.id),
+    discussionTopics: normalizeDiscussionTopic(data.plugin_configuration.discussion_topics),
```

Comment:
already discovered that this is breaking in test cases when discussion_topics are undefined